internal/pkg/term/selector/selector.go

```diff
@@ -23,6 +23,7 @@ import (
 const (
 	svcWorkloadType = "service"
 	jobWorkloadType = "job"
+	workloadType    = "workload"
```

Comment:
Maybe `wkldWorkloadType` to be more consistent 🤔 Also avoid conflating with the parameter name for `workloadType`.